Defra.Cdp.Backend.Api.IntegrationTests/Services/Deployments/DeploymentServiceTest.cs

```diff
@@ -0,0 +1,196 @@
+using Defra.Cdp.Backend.Api.IntegrationTests.Mongo;
+using Defra.Cdp.Backend.Api.Models;
+using Defra.Cdp.Backend.Api.Mongo;
+using Defra.Cdp.Backend.Api.Services.Aws.Deployments;
+using Defra.Cdp.Backend.Api.Services.Deployments;
+using Defra.Cdp.Backend.Api.Services.Github;
+using Defra.Cdp.Backend.Api.Services.Github.ScheduledTasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using NSubstitute;
+using NSubstitute.ReturnsExtensions;
+
+namespace Defra.Cdp.Backend.Api.IntegrationTests.Services.Deployments;
+
+public class DeploymentServiceTest : IClassFixture<MongoIntegrationTest>
+{
+    readonly MongoIntegrationTest Fixture;
+
+    readonly IRepositoryService repositoryService = Substitute.For<IRepositoryService>();
```

Comment:
Any reason to mock the database rather than use the Mongo from the integration test?